flexmeasures/data/models/reporting/tests/test_pandas_reporter.py

```diff
@@ -8,8 +8,9 @@
 def test_reporter(app, setup_dummy_data):
     s1, s2, reporter_sensor = setup_dummy_data
 
-    reporter_config_raw = dict(
-        beliefs_search_configs=[dict(sensor=s1.id), dict(sensor=s2.id)],
+    config = dict(
+        sensor=reporter_sensor.id,
+        input_variables=["sensor_1", "sensor_2"],
```

Comment:
The input variable names are already defined in the input dictionary as aliases, so I see an opportunity for simplification. Rather than having the CLI user define the variable names twice (both in the input and config), the config dictionary could just copy the aliases from the input dictionary.